straxen/plugins/bayes_peaklet_classification.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+import numba
+from scipy.special import logsumexp
+import strax
+import straxen
+
+
+class BayesPeakletClassification(strax.Plugin):
+    """
+    Bayes Peaklet classification
+    Returns the ln probability of a each event belonging to the S1 and S2 class.
+    Uses conditional probabilities and data parameterization learned from wfsim data.
+
+    :param peaklets: peaklets
+    :param waveforms: peaklets waveforms in PE/ns
+    :param quantiles: quantiles in ns
+    :returns: the ln probability of a each peaklet belonging to S1 and S2 class
+    """
+    provides = 'peaklet_classification_bayes'
+    depends_on = ('peaklets',)
+    __version__ = '0.0.1'
+    dtype = (strax.peak_interval_dtype
```

Comment:
maybe the strax.time_fields is more natural here?

Reply:
fixed